From 4d2a3bd7c127692dbf88bc606ac6a6a0f32714fd Mon Sep 17 00:00:00 2001
Message-Id: <4d2a3bd7c127692dbf88bc606ac6a6a0f32714fd.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Vidya Srinivas <vidya.srinivas@intel.com>
Date: Fri, 26 Sep 2014 11:56:58 +0530
Subject: [PATCH 301/312] Revert "REVERTME [VPG]: Update the primary plane on
 zorder change"

This reverts commit a74af1d987738ecfdac8158ec995908dd5f5556d.
Issue will be fixed with I7b9b8a11caa558a1add9dd65e80220ef7f2bfc79.

Issue: GMINL-1364
Change-Id: Id727ee70b80e0e5aecc80c5c03f84afbdb9dc0cb
Signed-off-by: Vidya Srinivas <vidya.srinivas@intel.com>
---
 drivers/gpu/drm/i915/intel_sprite.c |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_sprite.c b/drivers/gpu/drm/i915/intel_sprite.c
index 11c395d..cc52446 100644
--- a/drivers/gpu/drm/i915/intel_sprite.c
+++ b/drivers/gpu/drm/i915/intel_sprite.c
@@ -320,7 +320,6 @@ int i915_set_plane_zorder(struct drm_device *dev, void *data,
 
 	struct intel_crtc *intel_crtc =
 			to_intel_crtc(dev_priv->plane_to_crtc_mapping[pipe]);
-	struct drm_crtc *crtc = &intel_crtc->base;
 
 	s1_zorder = (order >> 3) & 0x1;
 	s1_bottom = (order >> 2) & 0x1;
@@ -375,10 +374,6 @@ int i915_set_plane_zorder(struct drm_device *dev, void *data,
 		intel_crtc->sprite1_alpha = true;
 	else
 		intel_crtc->sprite1_alpha = false;
-
-	dev_priv->display.update_primary_plane(crtc, crtc->primary->fb, crtc->x,
-		crtc->y);
-
 	return 0;
 }
 
-- 
1.7.9.5

